### PR TITLE
Feature/ui change terminate icon

### PR DESCRIPTION
**Pull request by longdogz**

### CODE REVIEW

File: modules/ui/app/scripts/services/UtilitiesFactory.js [hunk 1]
```diff
@@ -1002,16 +1002,22 @@ angular.module('wasabi.services').factory('UtilitiesFactory', ['Session', '$stat
...
                         break;
                     case 'terminated':
                         stateChange = 'terminate';
+                        title = 'Permanently Terminate Experiment';
                         break;
                 }
-                DialogsFactory.confirmDialog('Are you sure you want to ' + stateChange + ' the experiment ' + experiment.label + '?', 'Confirm State Change',
+                var msg = 'Are you sure you want to ' + stateChange + ' the experiment ' + experiment.label + '?';
+                if (state.toLowerCase() === 'terminated') {
+                    msg = 'Are you sure you want to <span style="font-weight: bold;">PERMANENTLY TERMINATE</span> the experiment ' + experiment.label + '?';
```
Comment (gittify) [inline modules/ui/app/scripts/services/UtilitiesFactory.js]: this double confirmation is a good idea 👍 
Will there be a help icon explaining a bit more about permanently terminate state...?

Comment (longdogz) [inline modules/ui/app/scripts/services/UtilitiesFactory.js]: There is currently a description of the terminate state on help accessible from the experiments list (icon next to State in the headers of the list).  Do you think that is sufficient?  (cc @calvinwma )?  Or do you think we should also put a help icon near the control on the Details dialog?

Comment (gittify) [inline modules/ui/app/scripts/services/UtilitiesFactory.js]: Just thought its easier on the dialog box but its optional....we can always add it later too

File: modules/ui/app/scripts/controllers/DialogModalCtrl.js [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 angular.module('wasabi.controllers')
     .controller('DialogModalCtrl',
-        ['$scope', '$modalInstance', 'options', '$timeout',
-            function ($scope, $modalInstance, options, $timeout) {
+        ['$scope', '$modalInstance', 'options', '$timeout', '$sce',
```
Comment (abhishekjain88) [inline modules/ui/app/scripts/controllers/DialogModalCtrl.js]: Suggest using the expanded version for the variable name "sce" just to make it more explanatory while reading code.

**Comment by sudheerkethireddy** [comment]
looks good :shipit:

File: modules/ui/app/scripts/controllers/DialogModalCtrl.js (hunk 2 above)
Comment (longdogz) [inline modules/ui/app/scripts/controllers/DialogModalCtrl.js]: That's a built-in variable from Angular JS (1.x).  I had to look up what it stands for.  😄 I don't believe they have implemented it as a longer named variable, but I'll add a comment, "Strict Contextual Escaping".

**Pull request merged**